File: MadLibs.rb
puts "Welcome to Troy's Mad Libs Adventure!"

puts "Enter a singular noun: "
A = gets.chomp
puts "Enter another singular noun: "
A2 = gets.chomp
puts"OK. This is the last singular noun you will have to input: "
A3 = gets.chomp
puts "Now, can you enter a plural noun: "
B = gets.chomp
puts "Please enter an adjective: "
C = gets.chomp
puts "One more adjective please: "
C2 = gets.chomp
puts "And another adjective: "
C3 = gets.chomp
puts "Enter Left or Right: "
D = gets.chomp.upcase

Left = "The "+A+ " turns the corner and sees another group of " + B + " and get captured.\nTHE END"
Right ="The "+A+ " runs into a " + C+" "+A2+" and a "+ C2+" "+A3+" who help fight off the "+B+". \nTHE END"


puts "One day a " + A + " was being chased by a group of " + C3 + " " + B + "."
puts'PRESS ENTER TO CONTINUE '
Z = gets
puts "The " + A + " ran to an intersection and decided to go " + D + "."
puts'PRESS ENTER TO CONTINUE '
Z = gets
if D == "LEFT"
	puts Left
else
	puts Right
end

File: MadLibs.py
def Left():
	print("The "+Noun1+ " turns the corner and sees another group of " + PluralNoun1 + " and get captured.\n THE END")
def Right():
	print("The "+Noun1+ " runs into a " + Adjective1 +" "+Noun2+" and a "+ Adjective2+" "+Noun3+" who help fight off the "+PluralNoun1+". \n THE END")

print ("Welcome to Troy's Mad Libs Adventure!")


Noun1 = input("Enter a singular noun: ")
Noun2 = input("Enter another singular noun: ")
Noun3 = input("OK. This is the last singular noun you will have to input: ")
PluralNoun1 = input("Now, can you enter a plural noun: ")
Adjective1 = input("Please enter an adjective: ") 
Adjective2 = input("One more adjective please: ")
Adjective3 = input("And another adjective: ")
Direction1 = input("Enter Left or Right: ").upper()


print("One day a " + Noun1 + " was being chased by a group of " + Adjective3 + " " + PluralNoun1 + ".")
input('PRESS ENTER TO CONTINUE ')
print("The " + Noun1 + " ran to an intersection and decided to go " + Direction1 + ".")
input('PRESS ENTER TO CONTINUE ')
if Direction1 == "LEFT":
	Left()
else:
	Right()	
